packages/slider/src/RangeSlider.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Range from './rc-slider/Range'
+import { space, color } from 'styled-system'
+import styleSlider from './styleSlider'
+
+const RangeSlider = styleSlider(Range)
+
+RangeSlider.defaultProps = {
+  allowCross: false,
+  color: 'blue'
```

Comment:
This should be `primary.base`

Reply:
Can that be done in a follow-up PR? This version of pcln-slider does not require DS 2.3.1 yet.

Reply:
Sounds good to me